uber/configspec.ini

```diff
@@ -1068,13 +1068,14 @@ __many__ = string
 
 [[badge]]
 attendee_badge  = string(default="Attendee")
-supporter_badge = string(default="Supporter")
```

Comment:
We haven't used this badge type in years, so I figured now was a good time to remove it. I also added the Contractor badge I've been promising people -- it doesn't do much yet, but it's there!